forch/forchestrator.py

```diff
@@ -94,6 +94,15 @@ def initialize(self):
         self._attempt_authenticator_initialise()
         self._process_static_device_placement()
         self._process_static_device_behavior()
+        if self._faucetizer:
+            faucetizer.write_behavioral_config(self._faucetizer, self._behavioral_config_file)
+
+        # wait for faucet to load config
+        while True:
+            time.sleep(1)
```

Comment:
increase timeout (10s?) and add logging message, so it's obvious if the system is stuck on this for whatever reason.

Reply:
Changed to 10s and added error logging